src/irr/asset/CGLIWriter.cpp

```diff
@@ -146,7 +155,7 @@ namespace irr
 				for (uint16_t layer = 0; layer < imageInfo.arrayLayers; ++layer)
 				{
 					const uint16_t gliLayer = layersFlag ? layer / 6 : 0;
```

Comment:
layer still wrong

Reply:
still wrong

Reply:
after https://github.com/buildaworldnet/IrrlichtBAW/pull/411/files/3e1da52a9ec2f98eb970baffdf29ea139282b499..426d151665e4292916c1c0a2fd69deaa3416fd05

now is correct.